Frontend/RecRock-Calendar-Frontend/src/app/generated clients/clients.ts

```diff
@@ -4,19 +4,23 @@
 // </auto-generated>
 //----------------------
 
+import { Injectable } from "@angular/core";
+
 /* tslint:disable */
 /* eslint-disable */
 // ReSharper disable InconsistentNaming
-
+@Injectable({
+  providedIn: 'root',
+})
 export class UsersClient {
     private http: { fetch(url: RequestInfo, init?: RequestInit): Promise<Response> };
     private baseUrl: string;
     protected jsonParseReviver: ((key: string, value: any) => any) | undefined = undefined;
 
-    constructor(baseUrl?: string, http?: { fetch(url: RequestInfo, init?: RequestInit): Promise<Response> }) {
-        this.http = http ? http : window as any;
-        this.baseUrl = baseUrl !== undefined && baseUrl !== null ? baseUrl : "http://localhost:6600";
-    }
+    constructor() {
+      this.http = window as any;
+      this.baseUrl = "http://localhost:6600";
```

Comment:
use environment variable instead of hardcoded url
https://angular.io/guide/build